File: Python/2019/Winning Score.py
# J1

Ascores = []
Bscores = []

for i in range(3):
    Ascores.append(int(input()))

Ascore = (Ascores[0]) * 3 + (Ascores[1]) * 2 + (Ascores[2]) * 1

for i in range(3):
    Bscores.append(int(input()))

Bscore = (Bscores[0]) * 3 + (Bscores[1]) * 2 + (Bscores [2]) * 1

if Ascore > Bscore:
    print('A')
elif Ascore == Bscore:
    print('T')
else:
    print('B')

File: C++/2019/WinningScore.cpp
// J1

#include <iostream>
using namespace std;

int main() {
    int apoints[3], bpoints[3];

    for (int a=0; a < 3; a++) {
        cin >> apoints[a];
    }

    for (int a=0; a < 3; a++) {
        cin >> bpoints[a];
    }
    
    int ascore = apoints[0] * 3 + apoints[1] * 2 + apoints[2];
    int bscore = bpoints[0] * 3 + bpoints[1] * 2 + bpoints[2];

    if (ascore > bscore) {
        cout << "A" << endl;
    }
    else if (bscore > ascore) {
        cout << "B" << endl;
    }
    else {
        cout << "T" << endl;
    }
}
